editor/src/components/code-editor/code-editor.tsx

```diff
@@ -182,7 +182,7 @@ export class CodeEditor extends React.Component<CodeEditorProps, CodeEditorState
             () => {
               this.save(this.state.code, true, {
                 message: 'File saved and formatted',
-                level: 'SUCCESS',
+                level: 'INFO',
```

Comment:
The idea is to use `SUCCESS` only for things where the outcome (or timing of outcome) is unexpected, e.g. adding a dependency / uploading an image that takes an unpredictable amount of time.

Reply:
Sounds good but in that case we should rename it to reflect that (e.g. to `ASYNC_COMPLETED` or something highlighting that it was triggered by a successful async side effect). Probably something for a separate PR